music_downloader.py

```diff
@@ -3,19 +3,28 @@
 import os
 import glob
 from bs4 import BeautifulSoup
-import urllib2
-from urllib import quote_plus as qp
+
+# Version compatiblity
+import sys
+isPython2 = True
+if (sys.version_info > (3,0)):
+	from urllib.request import urlopen
+	from urllib.parse import quote_plus as qp
+	isPython2 = False
```

Comment:
better to  put raw_input=input here, this way we can avoid another comparison we are doing at line 21.